sources/platform/integrations/ai/pinecone.md

```diff
@@ -20,11 +20,25 @@ The Apify integration for Pinecone enables you to export results from Apify Acto
 
 Before you begin, ensure that you have the following:
 
-- A [Pinecone database](https://www.pinecone.io/) set up.
+- A [Pinecone database](https://www.pinecone.io/) and index set up.
 - A Pinecone index created & Pinecone API token obtained.
 - An [OpenAI API key](https://openai.com/index/openai-api/) to compute text embeddings.
 - An [Apify API token](https://docs.apify.com/platform/integrations/api#api-token) to access [Apify Actors](https://apify.com/store).
 
+### How to setup Pinecone database and create an index
+
+1. Sign up or log in to your Pinecone account and click on "Create Index".
```

Comment:
I assume "Create Index" is  a UI element in Pinecone? If so please use bold to accentuate it instead of quotation

Reply:
ok, thanks!